tests/endtoend/blob_functions/blob_functions_stein/function_app.py

```diff
@@ -8,7 +8,7 @@
 
 import azure.functions as func
 
-app = func.FunctionApp()
+app = func.FunctionApp(http_auth_level=func.AuthLevel.ANONYMOUS)

```

Comment:
As discussed - moving `http_auth_level` to `auth_level` within `@app.route`? - All the places where you are adding `auth_level`

Reply:
Will fix in the next PR